# Vector of radii of detected landmarks
float64[] radii
# Vector of x-coordinates of detected landmarks
float64[] x_pts
# Vector of y-coordinates of detected landmarks
float64[] y_pts